docs/CONTRIBUTING.md

```diff
@@ -10,55 +10,68 @@ This allows us to consider and process your contribution as quickly and smoothly
-* **Screenshots:** If possible, make at least one screenshot of the bug and include it in your bug report. You can do this by simply dragging-and-dropping the image file into the template you're filling out. *Optional*, but very helpful for us.
+* **Screenshots:** If possible, make at least one screenshot of the bug and include it in your bug report (without exposing protected data). You can do this by simply dragging-and-dropping the image file into the template you're filling out. *Optional*, but very helpful for us.
 * **System Details:** Tell us which device you were using, on which SORMAS version the error occurred and, depending on whether you were using the mobile app or the web app, your Android version or web browser.
 * **Additional Information:** If there is anything else you want to add to your request, you can put it here. *Optional*.
 
+Before creating a bug report, please check the following rules:
+
+1. If something seems to be working correctly but does not necessarily match your expectations of how it should be working, please consider opening a change request instead. This applies also for slow performing features.
+2. If you have more than one possible bug - especially when you are tempted to create a list of independent findings in the **Bug Description** - then please document each bug as separate issue.
+3. Bugs are only valid to create if you can point to a released version where the problem is occuring, not on the version currently und development.
+
 ### Change Request
 
 Change requests cover features that are already part of SORMAS. This primarily includes aspects (or whole features) for which you would like to request an altered behavior, but also small extensions (e.g. additional values being added to a dropdown field).
 
-* **Feature Description:** Please describe the feature you would like us to change like it is in SORMAS right now. If it is about a relatively big or a very general feature, naming it briefly might be enough. If it is about a very specific detail of said feature, please try to be more specific.
-* **Problem Description:** Tell us why you want us to change the feature and what you believe is wrong or could be improved about the way it is currently designed.
-* **Proposed Change:** Describe in as much detail as possible how you would like us to change the feature and what the expected outcome of your request should look like.
-* **Possible Alternatives:** If possible, provide alternative solutions in case your proposed change can not be implemented for any reason. This is generally *optional* but might be requested at a later point in time.
+* **Problem Description:** Tell us why you want us to changeor extend an existing feature and what you believe is wrong or could be improved about the way it is currently designed.
```

Comment:
changeor -> change or

Reply:
Slightly adjusted style:
Tell us why you want us to change or extend an existing feature in SORMAS. Is there something working differently than you expect it? What can we improved about our current design?